js/apps/system/_admin/aardvark/APP/frontend/js/arango/arango.js

```diff
@@ -1233,8 +1233,8 @@
           validateInput: function () {
             return [
               {
-                rule: Joi.string().regex(/^(\/(APP[^\/]+|(?!APP)[a-zA-Z0-9_\-\/]+))+$/i),
-                msg: 'May not contain /APP'
+                rule: Joi.string().regex(/^(.(?!APP(\/|$)/i, {invert: true}),
```

Comment:
Unless I'm mistaken this is a syntax error (count the parens).
Also the leading paren is unnecessary.
Also this regex doesn't seem to work?
Any reason you don't simply split by `/` and check if any of the items in the array matches "APP"?

Reply:
ups, let me check! Yes that looks wrong.

Reply:
> Any reason you don't simply split by / and check if any of the items in the array matches "APP"?

That would also be possible, but I thought we want to stay to the regex stuff.